package.json

```diff
@@ -15,20 +16,18 @@
         "express": "4.13.3",
         "body-parser": "1.13.3",
         "connect-ensure-login": "0.1.1",
-        "express-handlebars": "1.1.0",
+        "express-handlebars": "3.0.0",
         "express-session": "1.11.3",
         "oauth2orize": "1.0.1",
         "passport": "0.2.1",
         "passport-local": "1.0.0",
         "passport-oauth2-client-password": "0.1.2"
     },
     "devDependencies": {
-        "node-jqunit": "1.1.4",
-        "grunt": "0.4.5",
-        "grunt-shell": "0.6.4",
-        "grunt-contrib-jshint": "0.9.0",
+        "grunt": "1.0.1",
+        "fluid-grunt-eslint": "18.1.1",
         "grunt-jsonlint": "1.0.4",
-        "grunt-gpii": "1.1.0"
+        "grunt-shell": "1.3.0"        
```

Comment:
Found this line with extra white spaces at the end.
I wonder if we can report this when passing jsonlint.